R/g.report.part4.R

```diff
@@ -76,6 +76,9 @@ g.report.part4 = function(datadir = c(), metadatadir = c(), loglocation = c(),
     }
     nightsummary2 = as.data.frame(do.call(rbind, lapply(fnames.ms4, myfun)), stringsAsFactors = FALSE)
     nightsummary2$night = as.numeric(gsub(" ", "", nightsummary2$night))
+    nightsummary2$calendar_date = as.Date(nightsummary2$calendar_date, format = "%d/%m/%Y")
+    nightsummary2$calendar_date = as.character(nightsummary2$calendar_date)
```

Comment:
Use `format()` for converting timestamps to character, this is safer as it provide more predictable output.